src/components/application_manager/test/commands/hmi/simple_response_from_hmi_test.cc

```diff
@@ -56,6 +56,10 @@
 #include "hmi/ui_alert_response.h"
 #include "hmi/ui_change_registration_response.h"
 #include "hmi/ui_delete_command_response.h"
+#include "hmi/ui_delete_submenu_response.h"
+#include "hmi/ui_end_audio_pass_thru_response.h"
+#include "hmi/ui_perform_audio_pass_thru_response.h"
+#include "hmi/ui_perform_interaction_response.h"
```

Comment:
@LevchenkoS please, check cd74ad6fe5e384f30160ca2e7309ff6bceda5bb7. I added/fixed all included commands to tests.